test/ts-node-unit-tests/benchmarks/DataPool-CSV-constructor.bench.ts

```diff
@@ -0,0 +1,66 @@
+import type { BenchmarkContext, BenchmarkResult } from '../benchmark';
+import { performance } from 'node:perf_hooks';
+import { join } from 'node:path';
+
+
+export const config = {
+    sizes: [100, 1000, 10_000, 100_000, 1_000_000, 2_500_000]
+};
+
+export function before(size: number) {
+    function generateCSV(rows:number, columns: number){
+        let csv = 'id';
+        for (let column = 0; column < columns; column++) {
+            csv += `,col${column}`;
+        }
+        csv += '\n';
+
+        for (let row = 0; row < rows; row++) {
+            csv += `${row}`;
+            for (let j = 0; j < columns; j++) {
+                csv += `,${Math.random()}`;
+            }
+            csv += '\n';
+        }
+
+        return csv;
+    }
+
+    return {
+        fileName: `${size}-rows.csv`,
+        func: generateCSV.bind(undefined, [size, 5])
```

Comment:
Shouldn't we also test it with other column values?

Reply:
Yes and no. For now this test is mostly to confirm that the DataPool constructor doesn't do any loading, so rows vs columns should not matter too much. But it might be neater with a square dataset 🤔 

I think I will add a test for the DataTable with increasing # of columns though 👍